content/en/docs/tasks/access-application-cluster/configure-dns-cluster.md

```diff
@@ -2,6 +2,7 @@
 title: Configure DNS for a Cluster
 weight: 120
 ---
+### Configure DNS for a Kubernetes Cluster
```

Comment:
This is displaying in the in-page TOC correctly as an H3 with an empty parent H2, hence the double bullets (I think).

Reply:
Makes sense. The original had no heading levels. Try adding the description without the heading? 

Seems to me that this page has other problems, too -- maybe file an issue for long-term solution? bc why are we linking to a README buried in the k/k repo in the first place?